### DISCUSSION TITLE
TypeError: expected string or bytes-like object, got 'NoneType'

### DISCUSSION THREAD

**Discussion by efdalkaya** [UMEP-dev/UMEP#688] [General]
Hi all. I have installed QGIS 3.40 version. Install UMEP plugin. I get a error "Couldn't load plugin 'UMEP' due to an error when calling its classFactory() method 
TypeError: expected string or bytes-like object, got 'NoneType' ".  How can solve the problem?

**Comment by biglimp** [comment]
We need some more info in order to help you. What operation system are you using? Also, make sure that you have the latest version of UMEP installed.

**Reply by efdalkaya** [reply]
Yesterday I am using QGIS 3.40.1 but now update 3.40.2. Solved the problem. I think problem pandas. When I installled UMEP plugin, i get a "not install Pandas" error.
